app/views/layouts/listnav/_container_route.html.haml

```diff
@@ -26,3 +26,13 @@
             = link_to("#{ui_lookup(:table => "container_project")}: #{@record.container_project.name}",
                 {:controller => "container_project", :action => 'show', :id => @record.container_project.id.to_s},
                 :title => _("Show this container service's parent %s") % ui_lookup(:table => "container_project"))
+        - if role_allows(:feature => "container_service_show_list")
+          - num_services = @record.number_of(:container_services)
```

Comment:
All this block is wrong as 1 route has only 1 service, so it should be more similar to the code right here above (`project`).